var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -177,28 +177,30 @@ def configure_args(self):
 
         return args
 
-    examples_src_dir = 'examples/CXX'
-
     @run_after('install')
     def cache_test_sources(self):
         """Copy the example source files after the package is installed to an
         install test subdirectory for use during `spack test run`."""
-        self.cache_extra_test_sources([self.examples_src_dir])
+        self.cache_extra_test_sources(join_path('examples', 'CXX'))
```

Comment:
Why remove `examples_src_dir` property that was used here and at line 187?

Reply:
Granted, it should've been defined as `join_path('examples', 'CXX')` instead of `examples/CXX`.

Reply:
It was a variable made outside of the method and I though it would be better just to hard code it, since it's only used twice.

Reply:
It's a property of the class that was deliberately created to reduce the potential introduction of errors for saving and using the examples.